client/data/help/use-help-search-query.ts

```diff
@@ -1,10 +1,12 @@
 import { useQuery } from '@tanstack/react-query';
 import apiFetch from '@wordpress/api-fetch';
+import { buildQueryString } from '@wordpress/url';
+import React from 'react';
 import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
 
 export interface SearchResult {
 	link: string;
-	title: string | React.ReactChild;
+	title: string | React.ReactNode;
```

Comment:
I wonder if we can do just `import type { ReactNode } from 'react'` instead of importing the entire `React`.

Reply:
Agreed. Fixed via https://github.com/Automattic/wp-calypso/pull/78729/commits/e035aa0ac493d88a68b90b3003c0cbf5c41ab3d7 – I also removed the `string |` part as `ReactNode` allows for strings anyway.